crawl-ref/source/dat/des/branches/orc.des

```diff
@@ -1399,6 +1399,433 @@ x1...1...1x
+.cI...Ic.
+.cc.T.cc.
+.........
+ENDMAP
+
+NAME:   nicolae_orc_big_mining_drill
+TAGS:   transparent
+DEPTH:  Orc
+ORIENT: east
+KITEM:  r = stone w:15 / large rock w:1
+KMONS:  F = orc warlord
+KMONS:  O = orc w:15 / orc priest w:5 / orc wizard w:5 / orc warrior w:5
+NSUBST: O = F / OOr
+SUBST:  - = x', ` = .x, r = ....r, ' = .$
+MAP
+         xxxxxxxxx
+  xxxxxxxxxxxxxxxxxxxxx
+  `````xxxxxxxxxx--xxxxxx
+ ......`xxxxxxxx-''---xxx
+.....O..`xxxxxxx..''''-xx
+....ccrrvrrrvrrrv..O'''-x
+....cvrOrvrOrvrOrv..9''-x
+...Ocbvbbbvbbbvbbbv.8''-x
+....cbrvrOrvrOrvrO..9''-x
+....ccrrvrrrvrrrv..O'''-x
+.....O.`xxxxxxxxx...''-xx
+ ``....`xxxxxxxxxx--''-xx
+ xx````xxxxxxxxxxxxx--xxx
+  xxxxxxxxxxxxxxxxxxxxxx
+       xxxxxxxxxx
+ENDMAP
+
+NAME:  nicolae_orc_combat_training
+TAGS:  transparent
+DEPTH: Orc
+# From the list of gear orc knights can get according to mon-gear.cc
+KITEM: R = arbalest / chain mail / plate armour / great sword / long sword / \
+           battle axe / war axe / great mace / dire flail / bardiche / glaive /\
+           broad axe / halberd
+KMONS: K = orc knight
+KMONS: O = orc warrior / orc w:2
+KMONS: D = training dummy
+SUBST: R = RRRRR., O = OOOOO.
+MAP
+ccccccccccccc
+ccccRRRRRcccc
+cc..........c
+cI.DO.DO.DO.c
+c...........c
+cI..........+
+cK.DO.DO.DO.+
+cI..........+
+c...........c
+cI.DO.DO.DO.c
+cc..........c
+ccccRRRRRcccc
+ccccccccccccc
+ENDMAP
+
+NAME:   nicolae_orc_crystal_seam
+TAGS:   no_exits
+DEPTH:  Orc
+: if crawl.one_chance_in( 5 ) then
+NSUBST: - = 2=@ / -
+: else
+NSUBST: b = 2=@ / bbbbbbbb-
+: end
+CLEAR:  -
+MAP
+   b
+ -b
+ b-  bb
+b  -b
+  bb-   bb
+ b   -bb
+    bb-   bbb
+  bb   -bb
+     bbb-    bbb
+   bb    -bbb
+       bbb-    bbbb
+    bbb    -bbb
+        bbbb-     bbbb
+     bbb     -bbbb
+          bbbb-
+      bbbb
+ENDMAP
+
+NAME:   nicolae_orc_found_a_spring
+TAGS:   transparent no_pool_fixup
+DEPTH:  Orc
+SUBST:  M = wx, ~ = Ww
+NSUBST: D = W / _, E = W / _, F = W / _, H = W / _, J = W / _, K = W / _
+CLEAR:  _
+MAP
+ xxxxxx..@
+xxxMMxx...   EWWWDxxxx
+xxMww~...   FWE DWWxxx
+xMwwww~WWJ HWF    xwwx
+xxMww~..JWWWH     xxxx
+xxxMMxx...
+ xxxxxx..@
+ENDMAP
+
+NAME:   nicolae_orc_ghostly_orcs
+TAGS:   transparent
+DEPTH:  Orc
+KMONS:  D = spectral orc / spectral orc warrior w:5
+NSUBST: ' = 3=D / 3=D. / $..
+SUBST:  - = x., ` = x'
+: dgn.delayed_decay(_G, 'D', 'orc corpse')
+MAP
+ xxxxxxxxx
+xxx-....xxx
+xx-.-...-xx
+x....-...-x
+@....-x..-x
+@.x--xx-''x
+xxxxxx`'''x
+xxxx`''''`x
+ xx`''''`xx
+  xx`''`xx
+   xxxxxxx
+ENDMAP
+
+# so that's what happened to the dwarves...
+NAME:   nicolae_orc_hostile_takeover
+TAGS:   transparent
+DEPTH:  Orc
+SUBST:  - = xxx'
+NSUBST: ' = 7=D / $..
+: dgn.delayed_decay(_G, 'D', 'dwarf corpse')
+MAP
+ xxxxxxxx
+xxxxx-''xx.
+xx--x-''.x.
+x-''xxx....
+x-''.....x.
+xx-x...xxxx
+xxxx...''-x
+xx-''xx''xx
+xx-''-x-xx
+ xxxxxxxx
+ENDMAP
+
+NAME:  nicolae_orc_mage_tower
+TAGS:  transparent
+DEPTH: Orc
+KITEM: d = any book
+KMONS: S = orc sorcerer
+KMONS: M = orc wizard
```

Comment:
Three free books is overly generous for one sorcerer and some harmless chaff in Orc, considering how many sorcerers Orc:2 might contain. Even with additional sorcerers there either should be a lower book count, a lower randbook numspell count, or the books shouldn't overlap much (sorcerers do summon, conjure, and hex all at once, after all).